Dockerfile

```diff
@@ -43,15 +43,12 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 
 RUN ln -s -f /usr/bin/python3 /usr/bin/python
 
-# Install NCCL 2.2
-RUN apt-get --no-install-recommends install -qq -y --allow-downgrades --allow-change-held-packages libnccl2=2.3.7-1+cuda10.0 libnccl-dev=2.3.7-1+cuda10.0
```

Comment:
Is this now part of the NVIDIA base image?

Reply:
It was there, just different version, as much as I remember.